iBurn/src/main/java/com/gaiagps/iburn/api/IBurnService.java

```diff
@@ -468,6 +468,10 @@ private void bindBaseValues(PlayaItem item, android.content.ContentValues values
             // Set and cache location for later use by events
 
             if (item.location != null) {
+                // https://github.com/iBurnApp/BlackRockCityPlanner/issues/6
+                if (!TextUtils.isEmpty(item.location.string)) {
+                    item.location.string = item.location.string.replace("None None", "");
```

Comment:
It should be fine but I used 

“ None None” with a space at the beginning

Reply:
Oh I see. I did test that it seemed to look ok but noted in case we see another issue.